package.json

```diff
@@ -31,8 +31,8 @@
   },
   "dependencies": {
     "@snyk/composer-lockfile-parser": "^1.4.1",
-    "@snyk/dep-graph": "^2.3.0",
-    "@snyk/rpm-parser": "3.0.0",
+    "@snyk/dep-graph": "^2.5.0",
+    "@snyk/rpm-parser": "git+ssh://git@github.com/snyk/rpm-parser#1f3cceebf59e9c0e082595d7339bd5277bcff567",
```

Comment:
@tommyknows Is this legit? If I understand correctly it's depends on the `rpm-parser` pull request? and after that it will be merged we should fix this version to the according `rpm-parser` version?